### DISCUSSION TITLE
Client rpc channel is not ready

### DISCUSSION THREAD

**Discussion by zhangminxiaozhang** [milvus-io/milvus#26653] [Q&A and General discussion]
io.milvus.exception.ClientNotConnectedException: Client rpc channel is not ready
` <groupId>io.milvus</groupId>
    <artifactId>milvus-sdk-java</artifactId>
    <version>2.2.10</version>`

**Comment by yhmo** [comment]
"Client rpc channel is not ready"
Possible reasons:
- The MilvusServiceClient failed to connect the milvus server. Connection parameters might be incorrect
- The MilvusServiceClient has been shutdown

**Reply by zhangminxiaozhang** [reply]
my configuration parameters are correct. 
I don’t know if the version in the dependency conflicts with something else.

**Reply by yhmo** [reply]
Try the Java SDK example first: https://github.com/milvus-io/milvus-sdk-java/blob/2.2/examples/main/java/io/milvus/GeneralExample.java

If the example is running well on your machine, compare the example code with yours.
